ui/cypress/e2e/onboarding.test.ts

```diff
@@ -5,6 +5,23 @@ interface TestUser {
   bucket: string
 }
 
+describe('Onboarding Redirect', () => {
+  let user: TestUser
+  beforeEach(() => {
+    cy.flush()
+
+    cy.fixture('user').then(u => {
+      user = u
+    })
+
+    cy.visit('/')
+  })
+  it('Can redirect to onboarding page', () => {
+    cy.get('[data-testid="init-step--head-main"]', {timeout: 5000})
```

Comment:
```suggestion
    cy.getByTestID("init-step--head-main")
```
You also shouldn't need the timeout as Cypress will already wait 30 seconds to find any element.

Reply:
got it